en/index.rst

```diff
@@ -53,7 +53,6 @@ FISCO BCOS is a reliable, secure, efficient and portable blockchain platform wit
    docs/enterprise_tools/index.md
    docs/sdk/sdk.md
    docs/browser/browser.md
-   docs/design/index.rst

```

Comment:
Why delete this?

Reply:
> Why delete this?

We don't check this, so hide it @qyan-dev